cpp2sky/config.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+message SegmentConfig {
```

Comment:
I feel these two belong to tracer.

Reply:
I'm worrying about to belong this on TracerConfig to be a confusion when integrating Envoy. (It won't use ClientConfig and TracerConfig, but SegmentConfig will be.)

Reply:
Oh, I see. Then good to me.